include/gz/sim/Link.hh

```diff
@@ -356,6 +357,27 @@ namespace gz
                                   const math::Vector3d &_torque,
                                   const math::Vector3d &_offset) const;
 
+      /// \brief Get the axis-aligned bounding box of the link.
+      /// It generates an axis-aligned bounding box for a link based on the
+      /// collision shapes attached to the it. The link bounding box is
+      /// generated by merging all the bounding boxes of the collision
+      /// geometries.
+      /// \param _ecm
```

Comment:
```suggestion
      /// \param _ecm Entity-component manager.
```

Reply:
Done in https://github.com/gazebosim/gz-sim/pull/2787/commits/7cbf8a30fe3014ca0fbc1b11c36573224160a3d7